OpenTextBC.js

```diff
@@ -0,0 +1,89 @@
+{
+	"translatorID": "8bdd3e33-87a0-488e-aa06-05bf87bea942",
+	"label": "openTextBC",
```

Comment:
This should be able to support more than just OpenTextBC! For instance, [here's a book from URegina](https://opentextbooks.uregina.ca/leadershipandinfluencingchangeinnursing/chapter/chapter-1-identifying-your-leadership-strengths-and-opportunities-for-growth/) being served by the same software. The application is called Pressbooks (some more users/"networks" listed [here](https://pressbooks.directory/))...